go.mod

```diff
@@ -1,10 +1,11 @@
 module github.com/giantswarm/cluster-test-suites
 
-go 1.20
```

Comment:
Please don't change the Go version.
Is there something specifically in Go 1.21 you require for this change? If not, please keep this separate.

Reply:
> Go 1.21 - Teleport API package requires Go >1.21

Sorry, I missed this. 

Could this please be done as an individual PR so that it can be easily tracked in the git history?

Reply:
Yes, good point, will take the Go version change separately.